specification/search/resource-manager/Microsoft.Search/preview/2024-03-01-preview/search.json

```diff
@@ -0,0 +1,3298 @@
+          "description": "The list of Private Endpoint connections."
+        },
+        "nextLink": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Request URL that can be used to query next page of private endpoint connections. Returned when the total number of requested private endpoint connections exceed maximum page size."
+        }
+      },
+      "description": "Response containing a list of Private Endpoint connections.",
+      "x-ms-external": true
+    },
+    "SemanticSearch": {
+      "type": "string",
+      "enum": [
+        "disabled",
+        "free",
+        "standard"
+      ],
+      "x-ms-enum": {
+        "name": "SearchSemanticSearch",
+        "modelAsString": true,
+        "values": [
+          {
+            "value": "disabled",
+            "name": "disabled",
+            "description": "Indicates that semantic reranker is disabled for the search service. This is the default."
+          },
+          {
+            "value": "free",
+            "name": "free",
+            "description": "Enables semantic reranker on a search service and indicates that it is to be used within the limits of the free plan. The free plan would cap the volume of semantic ranking requests and is offered at no extra charge. This is the default for newly provisioned search services."
+          },
+          {
+            "value": "standard",
+            "name": "standard",
+            "description": "Enables semantic reranker on a search service as a billable feature, with higher throughput and volume of semantically reranked queries."
+          }
+        ]
+      },
+      "description": "Sets options that control the availability of semantic search. This configuration is only possible for certain Azure AI Search SKUs in certain locations."
+    },
+    "SharedPrivateLinkResource": {
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/SharedPrivateLinkResourceProperties",
+          "description": "Describes the properties of a Shared Private Link Resource managed by the Azure AI Search service."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/Resource"
+        }
+      ],
+      "description": "Describes a Shared Private Link Resource managed by the Azure AI Search service."
+    },
+    "SharedPrivateLinkResourceProperties": {
+      "properties": {
+        "privateLinkResourceId": {
+          "type": "string",
+          "description": "The resource id of the resource the shared private link resource is for."
+        },
+        "groupId": {
+          "type": "string",
+          "description": "The group id from the provider of resource the shared private link resource is for."
+        },
+        "requestMessage": {
+          "type": "string",
+          "description": "The request message for requesting approval of the shared private link resource."
```

Comment:
2080 -- maybe just "The message for requesting approval"    to avoid word duplication in the sentence